portal/src/main/java/com/liferay/faces/portal/render/internal/DelayedPortalTagRenderer.java

```diff
@@ -128,20 +127,6 @@ protected void renderComponentMarkup(FacesContext facesContext, UIComponent uiCo
 
 			markup.append(preChildMarkup);
 
-			// Ensure that scripts are rendered at the bottom of the page.
-			String scripts = portalTagOutput.getScripts();
```

Comment:
Liferay will automatically relocate scripts during non-Ajax requests. During Ajax requests, scripts do not need to be relocated since jsf.js will strip them and run them last.